lib/jira.ts

```diff
@@ -193,8 +183,16 @@ export class Jira {
         return true;
     }
 
-    public static getFinalStatusNames(config: ProjectConfig): string[] {
-        return config.statuses.filter(status => status.isDone).map(status => status.name);
+    public static getFinalStatuses(statuses: Status[]): Status[] {
+        return statuses.filter(status => status.isDone);
+    }
+
+    public static isDoneStatus(statuses: Status[], statusToCheck: string): Boolean {
+        const matchingStatuses = Jira.getFinalStatuses(statuses).filter(
```

Comment:
A smaller nit: instead of `filter` and `length > 0`, this could be optimized to use `find` and `!== undefined`